addons/hw_posbox_homepage/controllers/main.py

```diff
@@ -47,6 +47,7 @@
 six_payment_terminal_template = jinja_env.get_template('six_payment_terminal.html')
 list_credential_template = jinja_env.get_template('list_credential.html')
 upgrade_page_template = jinja_env.get_template('upgrade_page.html')
+support_page_template = jinja_env.get_template('support.jinja2')
```

Comment:
```suggestion
support_page_template = jinja_env.get_template('support.html')
```

Reply:
https://stackoverflow.com/a/29603707/16356970 matter of taste, but IDE will likely recognised syntaxtically with a jinja extension rather than HTML (and does not change anything for the code)